pkgs/tools/text/grip-search/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, boost, pkgconfig, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "grip-search";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner = "sc0ty";
+    repo = "grip";
+    rev = "v${version}";
+    sha256 = "0bkqarylgzhis6fpj48qbifcd6a26cgnq8784hgnm707rq9kb0rx";
+  };
+
+  nativeBuildInputs = [ pkgconfig cmake ];
+
+  buildInputs = [ boost ];
+
+  patchPhase = ''
+    sed -i -e 's/"CUSTOM-BUILD"/${version}/' src/general/config.h
```

```suggestion
    substituteInPlace src/general/config.h --replace "CUSTOM-BUILD" "${version}"
```

This gives a warning, if the search string is not found.